sympy/matrices/expressions/tests/test_matmul.py

```diff
@@ -119,3 +119,9 @@ def test_collapse_MatrixBase():
 
 def test_refine():
     assert refine(C*C.T*D, Q.orthogonal(C)).doit() == D
+
+def test_issue_9503():
+    from sympy import I
+    a = symbols('a')
+    M = Matrix([[1, 2 + I], [3, 4]])
+    assert transpose(MatMul(a, M)) == MatMul(transpose(a), transpose(M))
```

Comment:
> first please properly specify which transpose you are talking about ? 

I was talking about the transpose of a scalar (in the complex module). There does not even seem to exist a useful way of defining the transpose of a scalar in the matrix module.